docs/patterns/single-new-eks-observability-accelerators/single-new-eks-awsnative-fargate-observability.md

```diff
@@ -0,0 +1,162 @@
+# Single New EKS Fargate Cluster AWS Native Observability Accelerator
+
+## Architecture
+
+The following figure illustrates the architecture of the pattern we will be deploying for Single EKS Fargate Cluster Native Observability pattern using AWS native tools such as CloudWatch Logs and Container Insights.
+
+![Architecture](../images/cloud-native-arch.png)
```

Comment:
Architecture Diagram is incorrect, CWADOT addon uses Prometheus receiver. Also diagram has no reference to Fargate.

Reply:
updated the diagram but mkdocs not detecting it. idk why